src/cijobs/project.json

```diff
@@ -15,11 +15,13 @@
     "netcoreapp1.0": {
         "dependencies": {
             "Microsoft.NETCore.App": {
-                "type": "platform",
                 "version": "1.0.0"
             }
         },
         "imports": "dnxcore50"
     }
+  },
+  "runtimes": {
```

Comment:
Yeah I was wondering about that. Technically it doesn't really matter what's in the repo since we have to build, replace binaries for win10-arm64, and copy to the arm64 share manually regardless. 

I'll revert the project.json change. Going from shared->standalone is a small enough change that I think the drift is acceptable in this case.